### PR TITLE
#115: Fix static variable name

### PR DESCRIPTION
**Pull request by Croway**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Oh, wow, thanks a lot for the quick PR! LGTM, just how about adding some unit tests for that new method?

**Comment by Croway** [comment]
Totally agree, I gathered most interesting event names in the csv file, seems all good to me except `G1MMU,G1MMU`, I could handle this situation (number followed by upper case letter) but I'm not sure this rule applies always

**Comment by gunnarmorling** [comment]
> seems all good to me except G1MMU,G1MMU

Yeah, so to be consistent with others, where there is an "_" after a number, it probably should be "G1_MMU" in this case. If it's doable without too much hassle, let's change it into that, otherwise, it's fine as-is.

The test class should have the common license header only (wondering in fact why this did not fail the build, there should be a plug-in for that). And could you squash the commits into one and prefix its message with "🐛 #115 Fix..." (Using [Gitmoji](https://gitmoji.dev/) conventions helps to auto-organize change notes). Thanks again!

File: jfrunit-events-generator/src/test/java/org/moditect/jfrunit/generator/PascalToSnakeCaseTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+public class PascalToSnakeCaseTest {
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/pascalToSnakeCaseTest.csv", numLinesToSkip = 1)
+    public void eventNameToSnakeCase(String input, String expected) {
```
Comment (gunnarmorling) [inline jfrunit-events-generator/src/test/java/org/moditect/jfrunit/generator/PascalToSnakeCaseTest.java:11]: Can we have a test for null input and empty string, too?

Comment (Croway) [reply jfrunit-events-generator/src/test/java/org/moditect/jfrunit/generator/PascalToSnakeCaseTest.java:11]: added
```
,
"",""
```
in csv for null and empty string respectively

**Comment by Croway** [comment]
> "bug #115 Fix..."

Ahah, interesting, I didn't know gitmoji, is the commit message ok?

**Comment by gunnarmorling** [comment]
> is the commit message ok?

Yepp, looking good now. Thanks a lot! Also logged #117 for adding the license header check, seems we forgot it in the course of converting the project into multiple modules.

**Pull request merged**